src/chrome/content/rules/3DCenterForum.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="3DCenter.org Forum">
+	<target host="www.forum-3dcenter.org" />
```

Comment:
Add
`<target host="forum-3dcenter.org" />`
`<rule from="^http://forum-3dcenter\.org/" to="https://www.forum-3dcenter.org/"/>`
`<rule from="^http:" to="https:"/>`
and remove the existing www.forum-3dcenter.org rule.